### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by newkek**
- IRC👎 / Slack👍
- DSE compatibility
- C* version compatibility
- License dates
- DSE driver messaging for DSE users

### CODE REVIEW

**Review by tolbertam — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -115,13 +119,20 @@ Other features are available only when using Apache Cassandra 2.0 or higher (e.g
 Trying to use these with a cluster running Cassandra 1.2 will result in 
 an [UnsupportedFeatureException](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/exceptions/UnsupportedFeatureException.java) being thrown.
 
-__Note__: DataStax products do not support big-endian systems.
+If using _DataStax Enterprise_, the [DataStax Enterprise Java driver][dse-driver] provides 
+more features and better compatibility.
+
+__Disclaimer__: Some _DataStax/DataStax Enterprise_ products might partially work on
```
Comment (newkek) [inline README.md:127]: just a suggestion

File: README.md [hunk 2]
```diff
@@ -132,7 +143,7 @@ If you are having issues connecting to the cluster (seeing `NoHostAvailableConne
 
 
 ## License
-Copyright 2012-2015, DataStax
+Copyright 2012-2017, DataStax
```
Comment (adutra) [inline README.md]: This needs to be updated to 2018.

**Comment by newkek** [comment]
I updated some content, I think this is ready to merge

**Review by tolbertam — APPROVED** [review]

**Pull request merged**